idl/github.com/uber/cadence/shared.thrift

```diff
@@ -368,6 +368,7 @@ struct WorkflowExecutionInfo {
   100: optional Memo memo
   101: optional SearchAttributes searchAttributes
   110: optional ResetPoints autoResetPoints
+  120: optional bool isWorkflowOpen
```

Comment:
Is there any customer asking for this? Can they just use `CloseStatus == Nil` for this?

I don't think we should add this field here, because:
first it is little bit redundant.
second WorkflowExecutionInfo is also used in visibility APIs (such as ListWorkflowExecutions), this change will be cause ListOpenWorkflows return workflow with `isWorkflowOpen` be false, which is confusing

Reply:
If possible, I would just ask customer to use `CloseStatus == Nil`

If not, I would add describe open workflow flag in DescribeWorkflowExecutionRequests to return empty resp if workflow already closed.